service/src/main/java/uk/nhs/adaptors/gp2gp/common/storageconnectors/StorageConnectorOptions.java

```diff
@@ -0,0 +1,6 @@
+package uk.nhs.adaptors.gp2gp.common.storageconnectors;
+
+public class StorageConnectorOptions {
+    public static final String S3 = "S3";
```

Comment:
Could we move these values to a config file?

Reply:
I think they are ok enumerated in code. You could consider changing this to a Java enum though.